src-ts/tools/learn/learn-lib/my-course-card/in-progress/InProgress.tsx

```diff
@@ -20,8 +19,10 @@ import { CurriculumSummary } from '../../curriculum-summary'
 import styles from './InProgress.module.scss'
 
 interface InProgressProps {
-    certification: LearnMyCertification
-    progress: LearnMyCertificationProgress
+    certification: LearnCertification
+    completed: number
```

Comment:
I'm assuming this is `percentCompleted` so I would recommend renaming this to be clearer.